tests/bluetooth/ctrl_isoal/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include_directories("./src")
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(bluetooth_ctrl_isoal)
+
+zephyr_library_include_directories(
+	${ZEPHYR_BASE}/subsys/bluetooth
```

Comment:
Isn't this already in the include path?

at least it looks that way from

zephyr/subsys/bluetooth/CMakeLists.txt

please update other tests as well if they have the same problem

Reply:
The build fails if this is removed. It does not appear to be in the include path.